rn-salesforce-chat.podspec

```diff
@@ -10,11 +10,11 @@ Pod::Spec.new do |s|
 
   s.authors      = package['author']
   s.homepage     = package['homepage']
-  s.platform     = :ios, "9.0"
+  s.platform     = :ios, "12.0"
```

Comment:
❓ Could this affect the iOS number version we are delivering the app currently (iOS 11)?